src/assets/stylesheets/utilities/_mixins.scss

```diff
@@ -34,19 +34,49 @@
     $column-width: (100% / $per-row);
 
     @media ( min-width: $min-width * $per-row ) {
-      min-width: calc(#{$column-width} - #{$gutter});
       flex: 1 0 calc(#{$column-width} - #{$gutter});
+      min-width: calc(#{$column-width} - #{$gutter});
     }
 
     @for $i from 2 through $per-row {
       $screen-width: ($min-width * $i) + ($gutter * $i);
       $column-width: (100% / $i);
 
       @media ( min-width: $screen-width ) {
-        max-width: calc(#{$column-width} - #{$gutter});
         flex: 1 0 calc(#{$column-width} - #{$gutter});
+        max-width: calc(#{$column-width} - #{$gutter});
```

Comment:
hahah I was getting a VIM linter error so I switched them...really dumb since it doesn't matter